### PR TITLE
affine args for filtering fixed

### PR DESCRIPTION
**Pull request by shaydeci**
For some reason these changes were dropped after merging master into the previous SSD PR (probably resolved conflicts with the lint PR wrongly by mistake).. https://github.com/Deci-AI/super-gradients/commit/02f449891ac17ef9a2f59c6214b075724d215fef#diff-5dd6cd98912e232235c9f2a845a7f875dad7ca7be3e26134c4e3e223d62699a2

### CODE REVIEW

File: src/super_gradients/training/transforms/transforms.py [hunk 1]
```diff
@@ -410,7 +410,7 @@ def __call__(self, sample: Union[dict, list]):
             all_samples = [sample] + sample["additional_samples"]
 
             for i_mosaic, mosaic_sample in enumerate(all_samples):
-                img, _labels = mosaic_sample["image"], mosaic_sample["target"]
+                img, _labels, = mosaic_sample["image"], mosaic_sample["target"]
```
Comment (ofrimasad) [inline src/super_gradients/training/transforms/transforms.py:413]: I think this was already fixed when @Louis-Dupont fixed lint

File: src/super_gradients/training/transforms/transforms.py [hunk 2]
```diff
@@ -487,18 +487,31 @@ class DetectionRandomAffine(DetectionTransform):
...
         from (shear, shear)
 
-    enable: (bool) whether to apply the below transform at all.
+     enable: (bool) whether to apply the below transform at all.
+
+     filter_box_candidates: (bool) whether to filter out transformed bboxes by edge size, area ratio, and aspect ratio (default=False).
+
+     wh_thr: (float) edge size threshold when filter_box_candidates = True (default=2)
+
+     ar_thr: (float) aspect ratio threshold filter_box_candidates = True (default=20)
```
Comment (ofrimasad) [inline src/super_gradients/training/transforms/transforms.py:496]: will it filter out boxes with higher or lower AR?

File: src/super_gradients/training/transforms/transforms.py [hunk 3]
```diff
@@ -487,18 +487,31 @@ class DetectionRandomAffine(DetectionTransform):
      shear: (Union[tuple, float]) degrees for random shear, when float the random values are drawn uniformly
         from (shear, shear)
 
-    enable: (bool) whether to apply the below transform at all.
+     enable: (bool) whether to apply the below transform at all.
+
+     filter_box_candidates: (bool) whether to filter out transformed bboxes by edge size, area ratio, and aspect ratio (default=False).
+
+     wh_thr: (float) edge size threshold when filter_box_candidates = True (default=2)
```
Comment (ofrimasad) [inline src/super_gradients/training/transforms/transforms.py:494]: write that boxes *smaller* than threshold will be filtered out

**Review by ofrimasad — COMMENTED** [review]
minor comments

File: src/super_gradients/training/transforms/transforms.py (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/training/transforms/transforms.py]: fixed https://github.com/Deci-AI/super-gradients/pull/294/commits/8bdf12438e74d3efb245b92baa1ff9ffef3c7306

File: src/super_gradients/training/transforms/transforms.py (hunk 2 above)
Comment (shaydeci) [reply src/super_gradients/training/transforms/transforms.py]: all thresholds clarified in docs https://github.com/Deci-AI/super-gradients/pull/294/commits/8bdf12438e74d3efb245b92baa1ff9ffef3c7306

File: src/super_gradients/training/transforms/transforms.py (hunk 3 above)
Comment (shaydeci) [reply src/super_gradients/training/transforms/transforms.py]: all thresholds clarified in docs https://github.com/Deci-AI/super-gradients/commit/8bdf12438e74d3efb245b92baa1ff9ffef3c7306

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**